web/app.py

```diff
@@ -6,11 +6,29 @@
 pint = UnitRegistry()
 
 
+def normalize_unit(language_code, unit):
+    if language_code[:2].lower() in {"cs", "sk"}:
+        spoon_units = {
+            "cl": ("ml", 6),  # čajová lžička
+            "CL": ("ml", 6),  # cL would be less-ambiguous
+            "ML": ("ml", 2),  # mL would be less-ambiguous
+            "pl": ("ml", 6),  # polévková lžíce
+            "PL": ("ml", 6),
```

Comment:
I think the conversion rates here are incorrect.  `pl` and `PL` should indeed be approximately mililiters-times-six, but the other units are all teaspoons/coffeespoons, and should be mililiters-times-two (I think?).

Reply:
Note: the choice of `6` and `2` here were chosen to match the [`frontend`](https://github.com/openculinary/frontend/tree/5418ec51ee0f7909d5b5edfff9658b296b52ca29) application's [`convert-units`](https://github.com/convert-units/convert-units/tree/93883046e43b5b0825489de076b9698de013503c) component.

Reply:
The selected resolution here (found in efbed02e3d88f11ebefe628bb3391e1d3af58012) is to re-use other already-defined unit conversions that the `pint` library is aware of.

We may need to check for unit definition and conversion discrepancies between `check-units` and `pint`.